pkg/test/resourcefixture/testdata/basic/sql/v1beta1/sqlinstance/mysqlinstance/_http.log

```diff
@@ -176,8 +177,8 @@ Grpc-Metadata-Content-Type: application/grpc
     "cert": "-----BEGIN CERTIFICATE-----\n-----END CERTIFICATE-----\n",
     "certSerialNumber": "0",
     "commonName": "common-name",
-    "createTime": "2024-06-19T22:54:22.309289161Z",
-    "expirationTime": "2025-06-19T22:54:22.309289161Z",
+    "createTime": "2024-06-20T23:54:19.971010960Z",
+    "expirationTime": "2025-06-20T23:54:19.971010960Z",
```

Comment:
@yuwenma should we normalize these timestamps?

Reply:
Looks like this was fixed in https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/2094